### PR TITLE
Display an error when trying to import in designer a table that's already imported

### PR DESCRIPTION
**Pull request by Tithugues**
It's currently possible to add many times the same table in the designer. This should not be allowed.

I used an existing message… Please tell me if a new one should be used and how to do it.

Thank you.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
LGTM :100:

File: js/messages.php [hunk 1]
```diff
@@ -751,6 +751,9 @@ function () {
 $js_messages['strU2FTimeout'] = __('Timed out waiting for security key activation.');
 $js_messages['strU2FError'] = __('Failed security key activation (%s).');
 
+/* Designer */
+$js_messages['strTableAlreadyExists'] = __('Table %s already exists!');
```
Comment (williamdes) [inline js/messages.php]:
```suggestion
$js_messages['strTableAlreadyExists'] = _pgettext('The table already exists in the designer and can not be added once more.', 'Table %s already exists!');
```

**Review by williamdes — CHANGES_REQUESTED** [review]
I added a suggestion to add context for translators, do you agree ?

File: js/messages.php (hunk 1 above)
Comment (Tithugues) [reply js/messages.php]: Honnestly, I don't really know the impact of this change. I looked at the `_pgettext` function, but don't really this the need for the context.
So I'll rely on you and commit your suggestion.

Comment (williamdes) [reply js/messages.php]: The impact is that the translators know what they are translating
https://github.com/phpmyadmin/phpmyadmin/blob/a2308ee965b70d77d74f3570f60cc6fba9580637/po/phpmyadmin.pot#L912
image
https://hosted.weblate.org/translate/phpmyadmin/4-9/fr/?checksum=&offset=1&q=Table&search=exact&type=translated&ignored=&date=&only_user=&exclude_user=

**Pull request merged**